js/script.js

```diff
@@ -0,0 +1,71 @@
+import speakersData from './speakersData.js';
+
+const speakers = speakersData();
+
+function markupSpeaker(speaker) {
+  return `<div class="speaker-card">
+            <div class="grid-bg"></div>
+            <img src="${speaker.img}" class="speaker-img" alt="${speaker.name}">
+            <div class="speaker-info">
+              <h3 class="speaker-name">${speaker.name}</h3>
+              <p class="speaker-title">${speaker.title}</p>
+              <hr class="sep-info">
+              <p class="speaker-bio">${speaker.bio}</p>
+            </div>
+          </div>`;
```

Comment:
- [ ] Please make your speaker card an `<article>` tag as it is much more semantically meaningful in this case than a `<div>`